dbuf-core/src/ast/parsed/location.rs

```diff
@@ -47,3 +68,48 @@ where
         self.start + self.length
     }
 }
+
+impl<Pos: Copy> Span for Location<Pos>
+where
+    Pos: Add<Offset, Output = Pos>,
+    Pos: Sub<Pos, Output = Offset>,
+{
+    type Context = ();
+    type Offset = Pos;
```

Comment:
Well, judging from the API, I think this should be
```rust
type Offset = Offset;
```
And we have to refactor the definition of `Location` into
```rust
pub struct Location<Module> {
    pub module: Module,
    pub start: Offset,
    pub length: Offset
}
```
To fit this into `Span` API.